card/mod/carrierwave/set/abstract/attachment/upload_cache.rb

```diff
@@ -48,6 +48,12 @@ def assign_attachment file, original_filename
   return unless (action = Card::Action.fetch(@action_id_of_cached_upload))
   upload_cache_card.delete_files_for_action action
   action.delete
+end
+
+# at some point uploaded files of canceled file card creation
+# should be deleted. We do this when ever an new file is created.
+event :clear_draft_files, :integrate_with_delay,
+      on: :create do
```

Comment:
any chance of race conditions here?  Should we test that the tmp files are more than a minute old, say?